ethers-etherscan/src/utils.rs

```diff
@@ -0,0 +1,61 @@
+use once_cell::sync::Lazy;
+use regex::Regex;
+use semver::Version;
+
+use crate::{EtherscanError, Result};
+
+static SOLC_BIN_LIST_URL: &str =
+    "https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/list.txt";
+
+static RE_SOLC_VERSION: Lazy<Regex> =
+    Lazy::new(|| Regex::new("soljson-v(?P<version>.*)\\.js").unwrap());
```

Comment:
looks like this can also be achieved via a simple trim_(start|end)_matches, then we don't need once_cell + regex

Reply:
good point 👍